Snakefile

```diff
@@ -22,23 +22,29 @@ rule all:
         GS.remote(f"{config['Gene2Phenotype']['outputBucket']}/gene2phenotype-{timeStamp}.json.gz"),
         GS.remote(f"{config['CRISPR']['outputBucket']}/crispr-{timeStamp}.json.gz"),
         GS.remote(f"{config['SysBio']['outputBucket']}/sysbio-{timeStamp}.json.gz"),
+        # GS.remote(f"{config['Phenodigm']['outputBucket']}/phenodigm-{timeStamp}.json.gz"),
+        # directory(GS.remote(f"{config['EPMC']['outputBucket']}/epmc-{timeStamp}")),
+        # directory(GS.remote(gs://genetics-portal-analysis/l2g-platform-export/data/genetics_portal_evidence.json.gz)),
```

Comment:
Could you comment a bit on these lines and why they're commented out? Are we planning to add them in the future?

Also, the PhenoDigm line seems to occur twice (one instances is commented out and another one is not)

Reply:
So all the green rows are added in the PR, however I'm not sure why the ones commented are remained commented. Fix them.